python/ql/src/experimental/semmle/python/frameworks/Yaml.qll

```diff
@@ -41,6 +41,9 @@ private module Yaml {
 /**
  * A call to `yaml.load`
  * See https://pyyaml.org/wiki/PyYAMLDocumentation (you will have to scroll down).
+ *
+ * This function was briefly thought safe until new exploits were found in 2020,
+ * see https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation for details.
  */
```

Comment:
```suggestion
 * This function was thought safe from the 5.1 release in 2017, when default loader was changed to FullLoader.
 * In 2020 new exploits were found, meaning it's not safe. Current plans is to change default to SafeLoader in release 6.0
 * (as explained in https://github.com/yaml/pyyaml/issues/420#issuecomment-696752389). 
 * Until 6.0 is released, we will mark `yaml.load` as possibly leading to arbitrary code execution.
 * See https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation for more details.
 */
```

and ACTUALLY, this comment should just go with the `unsafe()` predicate -- even if `yaml.load` was completely safe, we still want to model it as a deserialization/decoding/unmarshaling

Reply:
I will commit and move..

Reply:
Actually, I cannot commit, since it is outdated...